src/RamExistenceCheck.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Souffle - A Datalog Compiler
+ * Copyright (c) 2019, The Souffle Developers. All rights reserved
+ * Licensed under the Universal Permissive License v 1.0 as shown at:
+ * - https://opensource.org/licenses/UPL
+ * - <souffle root>/licenses/SOUFFLE-UPL.txt
+ */
+
+/************************************************************************
+ *
+ * @file RamExistenceCheck.h
+ *
+ * Get the index pattern of a RAM existence check
+ *
+ ***********************************************************************/
+
+#pragma once
+
+#include "RamAnalysis.h"
+#include "RamOperation.h"
```

Comment:
Add an include for SearchColumns
```suggestion
#include "RamTypes.h"
```